### PR TITLE
Metadata: Obsidian.Obsidian version 0.11.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21692)

### CODE REVIEW

File: manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 0.11.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
```
Comment (ItzLevvie) [inline manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml:12]:
```suggestion
  Scope: user
```
It's a `user` application and isn't `machine`.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml (hunk 1 above)
Comment (vedantmgoyal2009) [reply manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml:12]: It will fail again then.

Comment (ItzLevvie) [reply manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml:12]: We won't be able to accept the PR if it's set to `machine`.

File: manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 0.11.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (ItzLevvie) [inline manifests/o/Obsidian/Obsidian/0.11.0/Obsidian.Obsidian.installer.yaml:10]:
```suggestion
  Architecture: x64
```
Try it with x64.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**